emop-maven-plugin/src/main/java/edu/cornell/AffectedSpecsMojo.java

```diff
@@ -45,6 +46,26 @@ public class AffectedSpecsMojo extends ImpactedClassMojo {
     private static final int TRIMMED_SPEC_NAME_INDEX = 4;
     private static final int SPEC_INDEX_IN_MSG = 5;
 
+    /**
+     * The path that specify the Javamop Agent JAR file.
```

Comment:
The path that specify the Javamop Agent JAR file. --> The path to the Javamop Agent JAR file.